.github/workflows/ci.yml

```diff
@@ -87,6 +87,23 @@ jobs:
       - uses: EmbarkStudios/cargo-deny-action@v1
         with:
           command: check advisories
+          log-level: error
```

Comment:
By default the log level is warning. Most of the warnings we get are deprecated repo/unmaintained repos that we depend on indirectly. Setting it to error can reduce noise while notifying us if there is a security issue. I am open to suggestions if we should leave it to warning vs change it to error

Reply:
Hmm, I think you can add yanked/deprecated to an allow list, and I think that might be better? If I understand correctly this would also silence warnings about direct dependencies that are yanked/unmaintained, which seems undesirable.

Reply:
I think if we allow yanked/deprecated it means we allow it for both direct dependencies and indirect dependencies. I don't think cargo deny differentiate direct dependencies vs indirect dependencies